README.md
# joomla-quickstart

CHANGELOG.MD
# Change Log

## [0.6.0] - 2017-11-13

## Changed
- Upgrade Joomla version 3.7.3 to 3.8.2
- Update Alledia Framework 1.5.10 to 1.6.0
- Update Chronoform V5 5.0.14 to 5.0.16
- Update JCE Editor 2.6.16 to 2.6.20
- Update OSSystem 1.2.6 to 1.3.0
- Update CK Slideshow 1.4.44 to 1.4.46

## Added
- None

## Fixed
- None

## [0.5.0] - 2017-07-06

## Changed

- Joomla upgrade 3.6.5 to 3.7.3
- Update Alledia Framework 1.5.6 to 1.5.10
- Update Chronoform V5 5.0.12 to 5.0.14
- Update JCE Editor 2.6.1 to 2.6.16
- Update OSMap 4.2.6 to 4.2.12
- Update OSSystem 1.2.4 to 1.2.6
- Update CK Slideshow 1.4.40 to 1.4.44

## Added
- None

## Fixed
- None

## [0.4.0] - 2016-12-14

## Changed

- Joomla upgrade to 3.6.5
- Update JCE Editor 2.5.31 to 2.6.1
- Update OSMap 4.2.2 to 4.2.6
- Update Slideshow CK 1.4.39 to 1.4.40

## Added
- Add Facebook metadata publisher info in dt-clean

## Fixed
- None

## [0.3.0] - 2016-11-14

## Changed

- Joomla upgrade to 3.6.4
- Update browser-sync to 2.17.6 in dt-clean template
- Update AllediaFramework 1.5.1 to 1.5.6
- Update JCE Editor 2.5.19 to 2.5.31
- Update OSMap 4.1.3 to 4.2.2
- Update Slideshow CK 1.4.31 to 1.4.39
- Update PhocaGallery to 4.3.1

## Added
- google file webmasters tools

## Fixed

- detafile.js in dt-clean template
- Removed livereload js in index.php inside dt-clean
- Removed B2J override

## [0.2.1] - 2016-08-04

## Changed
- Joomla upgrade to 3.6.1

## [0.2.0] - 2016-07-27

## Added
- None

## Changed
- Update Aledia Framework 1.5.0 to 1.5.1
- Update JCE Editor 2.5.16 to 2.5.19
- Update OSMap 3.4.1 to 4.1.3
- Update ChronoForms v5 5.0.11 to 5.0.12
- Update Joomla 3.5.1 to 3.6.0

### Fixed
- Removed RokSprocket Module
- Removed Google Authenticator

## [0.1.0] - 2016-05-24

### Added
- detafile.js e package json em dt-clean
- Chrono form module v5
- Chrono form plugin v5
- This change log

### Changed
-  Update OSMap Component 3.4.0 to 3.4.1
- Changed logo admin
- Changed B2J Component for Chrono Forms  Component v5

### Fixed
- none


logs/joomla_update.php
#
#<?php die('Forbidden.'); ?>
#Date: 2016-04-14 14:41:28 UTC
#Software: Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT

#Fields: datetime	priority clientip	category	message
2016-04-14T14:41:28+00:00	INFO ::1	update	Atualização iniciada pelo usuário Super User (276). Versão antiga é 3.5.0.
2016-04-14T14:41:29+00:00	INFO ::1	update	Baixando arquivo de atualização de https://github.com/joomla/joomla-cms/releases/download/3.5.1/Joomla_3.5.x_to_3.5.1-Stable-Patch_Package.zip.
2016-04-14T14:41:32+00:00	INFO ::1	update	Arquivo Joomla_3.5.x_to_3.5.1-Stable-Patch_Package.zip baixado com sucesso.
2016-04-14T14:41:32+00:00	INFO ::1	update	Iniciando a instalação da nova versão.
2016-04-14T14:41:38+00:00	INFO ::1	update	Finalizando instalação.
2016-04-14T14:41:41+00:00	INFO ::1	update	Executou a consulta do arquivo 3.5.1-2016-03-25. Texto da consulta: ALTER TABLE `#__user_keys` MODIFY `user_id` varchar(150) NOT NULL;.
2016-04-14T14:41:41+00:00	INFO ::1	update	Executou a consulta do arquivo 3.5.1-2016-03-29. Texto da consulta: UPDATE `#__utf8_conversion` SET `converted` = 0  WHERE (SELECT COUNT(*) FROM `#_.
2016-04-14T14:41:41+00:00	INFO ::1	update	Excluindo arquivos e pastas removidas.
2016-04-14T16:49:55+00:00	INFO ::1	update	Atualização iniciada pelo usuário Super User (276). Versão antiga é 3.5.0.
2016-04-14T16:49:55+00:00	INFO ::1	update	Arquivo Joomla_3.5.x_to_3.5.1-Stable-Patch_Package.zip baixado com sucesso.
2016-04-14T16:49:56+00:00	INFO ::1	update	Iniciando a instalação da nova versão.
2016-04-14T16:50:00+00:00	INFO ::1	update	Finalizando instalação.
2016-04-14T16:50:02+00:00	INFO ::1	update	Executou a consulta do arquivo 3.5.1-2016-03-25. Texto da consulta: ALTER TABLE `#__user_keys` MODIFY `user_id` varchar(150) NOT NULL;.
2016-04-14T16:50:02+00:00	INFO ::1	update	Executou a consulta do arquivo 3.5.1-2016-03-29. Texto da consulta: UPDATE `#__utf8_conversion` SET `converted` = 0  WHERE (SELECT COUNT(*) FROM `#_.
2016-04-14T16:50:03+00:00	INFO ::1	update	Excluindo arquivos e pastas removidas.
2016-11-14T13:11:14+00:00	INFO ::1	update	Atualização iniciada pelo usuário Super User (276). Versão antiga é 3.6.1.
2016-11-14T13:11:16+00:00	INFO ::1	update	Baixando arquivo de atualização de https://downloads.joomla.org/cms/joomla3/3-6-4/joomla_3-6-4-stable-update_package-zip?format=zip&jcompat=my.zip.
2016-11-14T13:12:24+00:00	INFO ::1	update	Arquivo joomla_3-6-4-stable-update_package-zip?format=zip&jcompat=my.zip baixado com sucesso.
2016-11-14T13:12:24+00:00	INFO ::1	update	Iniciando a instalação da nova versão.
2016-11-14T13:19:22+00:00	INFO ::1	update	Iniciando a instalação da nova versão.
2016-11-14T13:23:38+00:00	INFO ::1	update	Finalizando instalação.
2016-11-14T13:23:48+00:00	INFO ::1	update	Executou a consulta do arquivo 3.6.3-2016-08-15. Texto da consulta: ALTER TABLE `#__newsfeeds` MODIFY `link` VARCHAR(2048) NOT NULL;.
2016-11-14T13:23:48+00:00	INFO ::1	update	Executou a consulta do arquivo 3.6.3-2016-08-16. Texto da consulta: INSERT INTO `#__postinstall_messages` (`extension_id`, `title_key`, `description.
2016-11-14T13:23:49+00:00	INFO ::1	update	Excluindo arquivos e pastas removidas.
2016-11-14T13:24:20+00:00	INFO ::1	update	Limpeza pós-instalação.
2016-11-14T13:24:20+00:00	INFO ::1	update	A atualização para a versão 3.6.4 está completa.
2016-12-14T19:04:38+00:00	INFO ::1	update	Atualização iniciada pelo usuário Super User (276). Versão antiga é 3.6.4.
2016-12-14T19:04:40+00:00	INFO ::1	update	Baixando arquivo de atualização de https://downloads.joomla.org/cms/joomla3/3-6-5/Joomla_3.6.5-Stable-Update_Package.zip.
2016-12-14T19:05:01+00:00	INFO ::1	update	Arquivo Joomla_3.6.5-Stable-Update_Package.zip baixado com sucesso.
2016-12-14T19:05:02+00:00	INFO ::1	update	Iniciando a instalação da nova versão.
2016-12-14T19:07:13+00:00	INFO ::1	update	Finalizando instalação.
2016-12-14T19:07:13+00:00	INFO ::1	update	Excluindo arquivos e pastas removidas.
2016-12-14T19:07:30+00:00	INFO ::1	update	Limpeza pós-instalação.
2016-12-14T19:07:30+00:00	INFO ::1	update	A atualização para a versão 3.6.5 está completa.
2017-07-06T12:18:16+00:00	INFO ::1	update	Atualização iniciada pelo usuário Super User (276). Versão antiga é 3.6.5.
2017-07-06T12:18:20+00:00	INFO ::1	update	Baixando arquivo de atualização de http://s3-us-west-2.amazonaws.com/joomla-official-downloads/joomladownloads/joomla3/Joomla_3.7.3-Stable-Update_Package.zip?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIZ6S3Q3YQHG57ZRA%2F20170706%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20170706T121813Z&X-Amz-Expires=60&X-Amz-SignedHeaders=host&X-Amz-Signature=e2677bd6257c650d28b87f3f0ded45e46e7b14ebf7b9c99dd84dbac214789168.
2017-07-06T12:18:42+00:00	INFO ::1	update	Arquivo Joomla_3.7.3-Stable-Update_Package.zip baixado com sucesso.
2017-07-06T12:18:44+00:00	INFO ::1	update	Iniciando a instalação da nova versão.
2017-07-06T12:22:14+00:00	INFO ::1	update	Finalising installation.
2017-07-06T12:22:16+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-08-06. Texto da consulta: INSERT INTO `#__extensions` (`extension_id`, `name`, `type`, `element`, `folder`.
2017-07-06T12:22:16+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-08-22. Texto da consulta: INSERT INTO `#__extensions` (`extension_id`, `name`, `type`, `element`, `folder`.
2017-07-06T12:22:22+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-08-29. Texto da consulta: CREATE TABLE IF NOT EXISTS `#__fields` (   `id` int(10) unsigned NOT NULL AUTO_I.
2017-07-06T12:22:24+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-08-29. Texto da consulta: CREATE TABLE IF NOT EXISTS `#__fields_categories` (   `field_id` int(11) NOT NUL.
2017-07-06T12:22:27+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-08-29. Texto da consulta: CREATE TABLE IF NOT EXISTS `#__fields_groups` (   `id` int(10) unsigned NOT NULL.
2017-07-06T12:22:29+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-08-29. Texto da consulta: CREATE TABLE IF NOT EXISTS `#__fields_values` (   `field_id` int(10) unsigned NO.
2017-07-06T12:22:29+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-08-29. Texto da consulta: INSERT INTO `#__extensions` (`extension_id`, `name`, `type`, `element`, `folder`.
2017-07-06T12:22:29+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-08-29. Texto da consulta: INSERT INTO `#__extensions` (`extension_id`, `name`, `type`, `element`, `folder`.
2017-07-06T12:22:29+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-09-29. Texto da consulta: INSERT INTO `#__postinstall_messages` (`extension_id`, `title_key`, `description.
2017-07-06T12:22:30+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-10-01. Texto da consulta: INSERT INTO `#__extensions` (`extension_id`, `name`, `type`, `element`, `folder`.
2017-07-06T12:22:31+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-10-02. Texto da consulta: ALTER TABLE `#__session` MODIFY `client_id` tinyint(3) unsigned DEFAULT NULL;.
2017-07-06T12:22:31+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-11-04. Texto da consulta: ALTER TABLE `#__extensions` CHANGE `enabled` `enabled` TINYINT(3) NOT NULL DEFAU.
2017-07-06T12:22:33+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-11-19. Texto da consulta: ALTER TABLE `#__menu_types` ADD COLUMN `client_id` int(11) NOT NULL DEFAULT 0;.
2017-07-06T12:22:33+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-11-19. Texto da consulta: UPDATE `#__menu` SET `published` = 1 WHERE `menutype` = 'main' OR `menutype` = '.
2017-07-06T12:22:34+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-11-21. Texto da consulta: ALTER TABLE `#__languages` DROP INDEX `idx_image`;.
2017-07-06T12:22:41+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-11-24. Texto da consulta: ALTER TABLE `#__extensions` ADD COLUMN `package_id` int(11) NOT NULL DEFAULT 0 C.
2017-07-06T12:22:41+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-11-24. Texto da consulta: UPDATE `#__extensions` AS `e1` INNER JOIN (SELECT `extension_id` FROM `#__extens.
2017-07-06T12:22:45+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2016-11-27. Texto da consulta: ALTER TABLE `#__modules` MODIFY `content` text NOT NULL DEFAULT '';.
2017-07-06T12:22:47+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_title` varchar(400) NOT NULL DEFAULT '.
2017-07-06T12:22:47+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_alias` varchar(400) CHARACTER SET utf8.
2017-07-06T12:22:54+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_body` mediumtext NOT NULL DEFAULT '';.
2017-07-06T12:22:55+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_checked_out_time` varchar(255) NOT NUL.
2017-07-06T12:23:02+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_params` text NOT NULL DEFAULT '';.
2017-07-06T12:23:02+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_metadata` varchar(2048) NOT NULL DEFAU.
2017-07-06T12:23:02+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_language` char(7) NOT NULL DEFAULT '';.
2017-07-06T12:23:02+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_publish_up` datetime NOT NULL DEFAULT .
2017-07-06T12:23:02+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_publish_down` datetime NOT NULL DEFAUL.
2017-07-06T12:23:08+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_content_item_id` int(10) unsigned NOT .
2017-07-06T12:23:15+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `asset_id` int(10) unsigned NOT NULL DEFAULT.
2017-07-06T12:23:25+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_images` text NOT NULL DEFAULT '';.
2017-07-06T12:23:36+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_urls` text NOT NULL DEFAULT '';.
2017-07-06T12:23:39+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_metakey` text NOT NULL DEFAULT '';.
2017-07-06T12:23:45+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_metadesc` text NOT NULL DEFAULT '';.
2017-07-06T12:23:45+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_xreference` varchar(50) NOT NULL DEFAU.
2017-07-06T12:23:59+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-08. Texto da consulta: ALTER TABLE `#__ucm_content` MODIFY `core_type_id` int(10) unsigned NOT NULL DEF.
2017-07-06T12:23:59+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-09. Texto da consulta: ALTER TABLE `#__categories` MODIFY `title` varchar(255) NOT NULL DEFAULT '';.
2017-07-06T12:24:12+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-09. Texto da consulta: ALTER TABLE `#__categories` MODIFY `description` mediumtext NOT NULL DEFAULT '';.
2017-07-06T12:24:17+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-09. Texto da consulta: ALTER TABLE `#__categories` MODIFY `params` text NOT NULL DEFAULT '';.
2017-07-06T12:24:17+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-09. Texto da consulta: ALTER TABLE `#__categories` MODIFY `metadesc` varchar(1024) NOT NULL DEFAULT '' .
2017-07-06T12:24:17+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-09. Texto da consulta: ALTER TABLE `#__categories` MODIFY `metakey` varchar(1024) NOT NULL DEFAULT '' C.
2017-07-06T12:24:17+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-09. Texto da consulta: ALTER TABLE `#__categories` MODIFY `metadata` varchar(2048) NOT NULL DEFAULT '' .
2017-07-06T12:24:17+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-09. Texto da consulta: ALTER TABLE `#__categories` MODIFY `language` char(7) NOT NULL DEFAULT '';.
2017-07-06T12:24:18+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-15. Texto da consulta: INSERT INTO `#__extensions` (`extension_id`, `name`, `type`, `element`, `folder`.
2017-07-06T12:24:18+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-17. Texto da consulta: UPDATE `#__menu`    SET `menutype` = 'main_is_reserved_133C585'  WHERE `client_i.
2017-07-06T12:24:18+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-17. Texto da consulta: UPDATE `#__modules`    SET `params` = REPLACE(`params`,'"menutype":"main"','"men.
2017-07-06T12:24:18+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-17. Texto da consulta: UPDATE `#__menu_types`    SET `menutype` = 'main_is_reserved_133C585'  WHERE `cl.
2017-07-06T12:24:19+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-17. Texto da consulta: UPDATE `#__menu`    SET `client_id` = 1  WHERE `menutype` = 'main';.
2017-07-06T12:24:19+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-17. Texto da consulta: UPDATE `#__menu`    SET `menutype` = 'main'  WHERE `client_id` = 1     AND `menu.
2017-07-06T12:24:19+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-17. Texto da consulta: UPDATE `#__menu`    SET `menutype` = 'main',        `client_id` = 1  WHERE `menu.
2017-07-06T12:24:19+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-17. Texto da consulta: DELETE FROM `#__menu_types`  WHERE `client_id` = 1    AND `menutype` IN ('main',.
2017-07-06T12:24:19+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-01-31. Texto da consulta: INSERT INTO `#__extensions` (`extension_id`, `name`, `type`, `element`, `folder`.
2017-07-06T12:24:19+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-02-02. Texto da consulta: INSERT INTO `#__extensions` (`extension_id`, `name`, `type`, `element`, `folder`.
2017-07-06T12:24:20+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-02-15. Texto da consulta: ALTER TABLE `#__redirect_links` MODIFY `comment` varchar(255) NOT NULL DEFAULT '.
2017-07-06T12:24:20+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-02-17. Texto da consulta: ALTER TABLE `#__contact_details` MODIFY `name` varchar(255) NOT NULL;.
2017-07-06T12:24:20+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-02-17. Texto da consulta: ALTER TABLE `#__contact_details` MODIFY `alias` varchar(400) CHARACTER SET utf8m.
2017-07-06T12:24:20+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-02-17. Texto da consulta: ALTER TABLE `#__contact_details` MODIFY `sortname1` varchar(255) NOT NULL DEFAUL.
2017-07-06T12:24:20+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-02-17. Texto da consulta: ALTER TABLE `#__contact_details` MODIFY `sortname2` varchar(255) NOT NULL DEFAUL.
2017-07-06T12:24:20+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-02-17. Texto da consulta: ALTER TABLE `#__contact_details` MODIFY `sortname3` varchar(255) NOT NULL DEFAUL.
2017-07-06T12:24:25+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-02-17. Texto da consulta: ALTER TABLE `#__contact_details` MODIFY `language` varchar(7) NOT NULL;.
2017-07-06T12:24:25+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-02-17. Texto da consulta: ALTER TABLE `#__contact_details` MODIFY `xreference` varchar(50) NOT NULL DEFAUL.
2017-07-06T12:24:28+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-03-03. Texto da consulta: ALTER TABLE `#__languages` MODIFY `asset_id` int(10) unsigned NOT NULL DEFAULT 0.
2017-07-06T12:24:31+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-03-03. Texto da consulta: ALTER TABLE `#__menu_types` MODIFY `asset_id` int(10) unsigned NOT NULL DEFAULT .
2017-07-06T12:24:31+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-03-03. Texto da consulta: ALTER TABLE  `#__content` MODIFY `xreference` varchar(50) NOT NULL DEFAULT '';.
2017-07-06T12:24:31+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-03-03. Texto da consulta: ALTER TABLE  `#__newsfeeds` MODIFY `xreference` varchar(50) NOT NULL DEFAULT '';.
2017-07-06T12:24:32+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-03-09. Texto da consulta: UPDATE `#__categories` SET `published` = 1 WHERE `alias` = 'root';.
2017-07-06T12:24:32+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-03-09. Texto da consulta: UPDATE `#__categories` AS `c` INNER JOIN ( 	SELECT c2.id, CASE WHEN MIN(p.publis.
2017-07-06T12:24:33+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-03-09. Texto da consulta: UPDATE `#__menu` SET `published` = 1 WHERE `alias` = 'root';.
2017-07-06T12:24:33+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-03-09. Texto da consulta: UPDATE `#__menu` AS `c` INNER JOIN ( 	SELECT c2.id, CASE WHEN MIN(p.published) >.
2017-07-06T12:24:38+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-03-19. Texto da consulta: ALTER TABLE `#__finder_links` MODIFY `description` text;.
2017-07-06T12:24:38+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-04-10. Texto da consulta: INSERT INTO `#__postinstall_messages` (`extension_id`, `title_key`, `description.
2017-07-06T12:24:39+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.0-2017-04-19. Texto da consulta: UPDATE `#__extensions` SET `params` = '{"multiple":"0","first":"1","last":"100",.
2017-07-06T12:24:46+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.3-2017-06-03. Texto da consulta: ALTER TABLE `#__menu` MODIFY `checked_out_time` datetime NOT NULL DEFAULT '0000-.
2017-07-06T12:24:48+00:00	INFO ::1	update	Deleting removed files and folders.
2017-07-06T12:25:33+00:00	INFO ::1	update	Cleaning up after installation.
2017-07-06T12:25:33+00:00	INFO ::1	update	Update to version 3.7.3 is complete.
2017-11-13T20:47:09+00:00	INFO ::1	update	Atualização iniciada pelo usuário Super User (276). Versão antiga é 3.7.3.
2017-11-13T20:47:16+00:00	INFO ::1	update	Baixando arquivo de atualização de https://s3-us-west-2.amazonaws.com/joomla-official-downloads/joomladownloads/joomla3/Joomla_3.8.2-Stable-Update_Package.zip?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIZ6S3Q3YQHG57ZRA%2F20171113%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20171113T204738Z&X-Amz-Expires=60&X-Amz-SignedHeaders=host&X-Amz-Signature=c56388820afa766ef3af4a6e3d6c2980d92c9b30cc90af64d253d2d447e116f4.
2017-11-13T20:47:35+00:00	INFO ::1	update	Arquivo Joomla_3.8.2-Stable-Update_Package.zip baixado com sucesso.
2017-11-13T20:47:36+00:00	INFO ::1	update	Iniciando a instalação da nova versão.
2017-11-13T20:52:56+00:00	INFO ::1	update	Finalizando instalação.
2017-11-13T20:52:57+00:00	INFO ::1	update	Executou a consulta do arquivo 3.7.4-2017-07-05. Texto da consulta: DELETE FROM `#__postinstall_messages` WHERE `title_key` = 'COM_CPANEL_MSG_PHPVER.
2017-11-13T20:52:58+00:00	INFO ::1	update	Executou a consulta do arquivo 3.8.0-2017-07-28. Texto da consulta: ALTER TABLE `#__fields_groups` ADD COLUMN `params` TEXT  NOT NULL  AFTER `orderi.
2017-11-13T20:52:59+00:00	INFO ::1	update	Executou a consulta do arquivo 3.8.0-2017-07-31. Texto da consulta: INSERT INTO `#__extensions` (`extension_id`, `package_id`, `name`, `type`, `elem.
2017-11-13T20:52:59+00:00	INFO ::1	update	Executou a consulta do arquivo 3.8.2-2017-10-14. Texto da consulta: ALTER TABLE `#__content` ADD INDEX `idx_alias` (`alias`(191));.
2017-11-13T20:52:59+00:00	INFO ::1	update	Excluindo arquivos e pastas removidas.
2017-11-13T20:53:23+00:00	INFO ::1	update	Limpeza pós-instalação.
2017-11-13T20:53:23+00:00	INFO ::1	update	A atualização para a versão 3.8.2 está completa.
